x-pack/legacy/plugins/ml/server/models/data_frame_analytics/analytics_audit_messages.ts

```diff
@@ -37,6 +37,11 @@ export function analyticsAuditMessagesProvider(callWithRequest: callWithRequestT
                   level: 'activity',
                 },
               },
+              must: {
+                term: {
+                  job_type: 'data_frame_analytics',
```

Comment:
Looks like a `term` query like this also needs to be added to the query used to populate the anomaly detection job messages. Here I created an analytics job with the same ID as an anomaly detection job, and the analytics job messages are showing up in the anomaly detection job:

![image](https://user-images.githubusercontent.com/7405507/68776516-1e000e00-0628-11ea-8fb5-b48552574bb9.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/50468/commits/a01112ff400cf6162c0fb87a4a006cad1a3f19a5